ROMFS/px4fmu_common/init.d/rcS

```diff
@@ -804,9 +840,14 @@ then
 	# Try to get an USB console
 	nshterm /dev/ttyACM0 &
 else
-	mavlink start -r 800000 -d /dev/ttyACM0 -m config -x
+	# Don't start mavlink in passthrough mode because we currently can't seem to stop it.
```

Comment:
mavlink stop-all fails for the USB one. Therefore don't start it at all... It's a hack.